### DISCUSSION TITLE
forum/

### DISCUSSION THREAD

**Discussion by giscus** [Wedot-Engine/Wedot-Engine.github.io#1] [Announcements]
# forum/

A temporary forum page for WeDot Engine community

https://wedot-engine.github.io/forum/

**Comment by PJ-568** [comment]
Hi! Welcome! The WeDot Forum is currently under construction. Please feel free to ask any question.

![Alt](https://repobeats.axiom.co/api/embed/e8d7fcfa4529d0f4a1a8383ced8d3cd7d2f50099.svg "Repobeats analytics image")

:-)

**Comment by WDK-YYDS** [comment]
王大可到此一游

**Comment by RainyGrass** [comment]
太棒了，怎么说，希望越来越好！

**Comment by mochenrqt** [comment]
加油！支持国产引擎~

**Comment by 996666925** [comment]
张三到此一游
